### DISCUSSION TITLE
How to test quic datagram?

### DISCUSSION THREAD

**Discussion by zhanghenan-dev** [microsoft/msquic#2267] [Q&A] [answered]
Is there a demo or sample to interact quic datagram? 
I found the interface "DatagramSend", but can't find a demo like quicsample(start a server and client) to use it.
Please tell me how to use ? Or I need to write my own demo ?

**Comment by nibanks** [comment, marked as answer]
No, we don't have a simple app that leverages the DatagramSend API yet.

**Reply by zhanghenan-dev** [reply]
Thanks, I got it.
